Core/src/org/sleuthkit/autopsy/centralrepository/eventlisteners/IngestEventsListener.java

```diff
@@ -52,7 +52,7 @@
     private static final Logger LOGGER = Logger.getLogger(EamArtifact.class.getName());
 
     final Collection<String> addedCeArtifactTrackerSet = new LinkedHashSet<>();
-
+    private static long ingestJobCounter = 0;
```

Comment:
Should ingestJobCounter be volatile?

Reply:
I was under the impression that since all access to it was in synchronized methods, so that they can be safely accessed from different threads, that making it volatile would not accomplish any additional safeguarding.  But perhaps I am missing something.

Reply:
Yeah I think you're right. Just wanted to be sure.

Reply:
This not really an ingest job counter, is it? I think it would be less confusing for future maintainers if we called it an ingest module instance count instead.